src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,491 @@
+package strawman
+package collection.mutable
+
+import scala._
```

Comment:
Why do you need this import? We added the `Yno-predef` flag to not have our scope polluted with `scala.List` or other default collection-esque imports from `scala`.